### PR TITLE
Adding libtensorlight

### PR DESCRIPTION
**Pull request by maxscheurer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by maxscheurer** [comment]
@conda-forge/help-c-cpp
Is there any way to download the packages built on Azure for running some local tests? Thanks 😊

File: recipes/libtensorlight/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+{% set sha256 = "1ec596bbd160b4df413bccb42e88d876e098aa62899f6b1bb50003d5552bf9cb" %}
+{% set build = 0 %}
+
+package:
+  name: libtensorlight
+  version: {{ version }}
+
+source:
+  url: https://github.com/adc-connect/libtensor/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/libtensorlight/meta.yaml:11]: Please write it directly here

File: recipes/libtensorlight/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+build:
+  number: {{ build }}
```
Comment (xhochy) [inline recipes/libtensorlight/meta.yaml:14]: Keep this variable free otherwise the bot will have issues

File: recipes/libtensorlight/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      - mkl 
+  commands:
+      - test -f $PREFIX/lib/libtensorlight$SHLIB_EXT
+      - ldd -v $PREFIX/lib/libtensorlight$SHLIB_EXT                 # [linux]
+      - otool -L $PREFIX/lib/libtensorlight$SHLIB_EXT               # [osx]
+      - conda inspect linkages --show-files --groupby=dependency libtensorlight
+      - conda inspect objects -p $PREFIX $PKG_NAME                  # [osx]
+
+outputs:
+  - name: libtensorlight
```
Comment (xhochy) [inline recipes/libtensorlight/meta.yaml:44]: If this is a single output, please don't use outputs

File: recipes/libtensorlight/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    - llvm-openmp            # [osx]
+    - libgomp                # [linux]
+    - cmake >=3.2
+    - ninja
+
+  host:
+    - mkl-devel
+
+  run:
+    - libgomp                # [linux]
```
Comment (xhochy) [inline recipes/libtensorlight/meta.yaml:31]: Should not be needed

```suggestion
```

File: recipes/libtensorlight/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+test:
+  requires:
+      - mkl 
+  commands:
+      - test -f $PREFIX/lib/libtensorlight$SHLIB_EXT
+      - ldd -v $PREFIX/lib/libtensorlight$SHLIB_EXT                 # [linux]
+      - otool -L $PREFIX/lib/libtensorlight$SHLIB_EXT               # [osx]
+      - conda inspect linkages --show-files --groupby=dependency libtensorlight
+      - conda inspect objects -p $PREFIX $PKG_NAME                  # [osx]
```
Comment (xhochy) [inline recipes/libtensorlight/meta.yaml:41]: Linkages are already checked by conda-build nowadays 
```suggestion
```

File: recipes/libtensorlight/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+  host:
+    - mkl-devel
+
+  run:
+    - libgomp                # [linux]
+
+test:
+  requires:
+      - mkl
```
Comment (xhochy) [inline recipes/libtensorlight/meta.yaml:35]: If it is needed here, should it be in run?

**Comment by xhochy** [comment]
> @conda-forge/help-c-cpp
> Is there any way to download the packages built on Azure for running some local tests? Thanks 😊 

Yes, you can download them there as artifacts.

File: recipes/libtensorlight/meta.yaml (hunk 3 above)
Comment (maxscheurer) [reply recipes/libtensorlight/meta.yaml]: When using `files` directly, I get "The top level meta key files is unexpected", that's why I used `outputs`.

File: recipes/libtensorlight/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libtensorlight$SHLIB_EXT
+
+files:
+  - include/libtensor
+  - include/libutil
+  - lib/libtensor*
+  - lib/pkgconfig/libtensorlight.pc
```
Comment (xhochy) [inline recipes/libtensorlight/meta.yaml:39]: You should be able to omit these completely 

```suggestion
```

File: recipes/libtensorlight/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+about:
+  home: https://github.com/epifanovsky/libtensor
+  dev_url: https://github.com/adc-connect/libtensor
+  license: BSD-1-Clause
+  license_family: Other
+  license_file: {{ SRC_DIR }}/LICENSE
```
Comment (xhochy) [inline recipes/libtensorlight/meta.yaml:46]:
```suggestion
  license_file: LICENSE
```

**Comment by maxscheurer** [comment]
Not sure if this is needed, but just to be sure:
@conda-forge/staged-recipes ready for review! 🚀

File: recipes/libtensorlight/meta.yaml [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+
+package:
+  name: libtensorlight
+  version: {{ version }}
+
+source:
+  url: https://github.com/adc-connect/libtensor/archive/v{{ version }}.tar.gz
+  sha256: 1ec596bbd160b4df413bccb42e88d876e098aa62899f6b1bb50003d5552bf9cb
+
+build:
```
Comment (xhochy) [inline recipes/libtensorlight/meta.yaml:11]: This also needs a fitting run_exports.

**Review by xhochy — COMMENTED** [review]
One comment, then this is good to go.

File: recipes/libtensorlight/meta.yaml (hunk 9 above)
Comment (maxscheurer) [reply recipes/libtensorlight/meta.yaml:11]: I'm always uncertain how to specify `run_exports`, also after re-reading [docs](https://conda-forge.org/docs/maintainer/pinning_deps.html#specifying-run-exports) again...

Would
```yaml
run_exports:
    - {{ pin_subpackage('libtensorlight', max_pin='x.x') }}
```
be fitting?

**Pull request merged**